.clusterfuzzlite/build.sh

```
@@ -0,0 +1,126 @@
+EOF
+
+# ── Compile standalone fuzz targets against the project classpath ──
+BUILD_CP="target/classes:target/test-classes:target/deps/*"
+javac -cp "${BUILD_CP}" \
+    -d $OUT \
+    $SRC/PathNavigatorFuzzer.java \
+    $SRC/MatchingUtilitiesFuzzer.java
+
+# ── Generate Jazzer wrapper scripts ──
+for fuzzer in PathNavigatorFuzzer MatchingUtilitiesFuzzer; do
+    cat > $OUT/${fuzzer} << WRAPPER
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)